base/internal/message.cpp

```diff
@@ -1,36 +1,7 @@
 #include "message.hpp"
 
-#include "std/target_os.hpp"
-
 #include <utf8/unchecked.h>
 
-std::string DebugPrint(std::string const & t)
-{
-#ifdef OMIM_OS_WINDOWS
```

Comment:
@Osyotr hope that 2010 year workaround is not needed anymore? Are utf8 strings printed fine now?

Reply:
I've also removed it so should be fine.